e2e/specs/st_map.spec.js

```diff
@@ -19,43 +19,44 @@ describe("st.map", () => {
     cy.loadApp("http://localhost:3000/");
 
     // Wait for all the maps to be loaded.
-    cy.get(".element-container .stDeckGlJsonChart").should("have.length", 7);
+    cy.get(".element-container .stDeckGlJsonChart").should("have.length", 4);
   });
 
-  it("displays 7 maps", () => {
-    cy.get(".element-container .stDeckGlJsonChart").should("have.length", 7)
+  it("displays 4 maps", () => {
+    cy.get(".element-container .stDeckGlJsonChart").should("have.length", 4);
   });
 
-  it("displays 7 zoom buttons", () => {
-    cy.get(".element-container .zoomButton").should("have.length", 7)
-  })
-
-  it("displays warning about data being capped", () => {
-    cy.get("div [data-testid='stCaptionContainer']")
-      .should("have.length", 2)
-
-    // Check warning body.
-    cy.getIndexed("div [data-testid='stCaptionContainer']", 0)
-      .should("contain", "⚠️ Showing only 10k rows. Call collect() on the dataframe to show more.")
-    cy.getIndexed("div [data-testid='stCaptionContainer']", 1)
-      .should("contain", "⚠️ Showing only 10k rows. Call collect() on the dataframe to show more.")
-  })
+  it("displays 4 zoom buttons", () => {
+    cy.get(".element-container .zoomButton").should("have.length", 4);
+  });
 
   it("displays basic and complex maps correctly", () => {
-    cy.get(".mapboxgl-canvas")
+    cy.get(".mapboxgl-canvas");
     // Adding a sufficient wait to ensure the map fully loads before taking the snapshot
-    cy.wait(10000)
-    cy.get(".element-container:has(.stDeckGlJsonChart)", { waitForAnimations: true })
-      .eq(5).matchImageSnapshot("stDeckGlJsonChart")
-    cy.get(".element-container:has(.stDeckGlJsonChart)", { waitForAnimations: true })
-      .eq(6).matchImageSnapshot("stDeckGlJsonChart-complexMap")
+    cy.wait(10000);
+    cy.get(".element-container:has(.stDeckGlJsonChart)", {
+      waitForAnimations: true,
+    })
+      .eq(2)
```

Comment:
In the future we should use constants (e.g. `expected_map_calls = 2`) more thoroughly instead of using magic numbers, as it would help better to understand why this number decreased (the amount of calls to `st.map` reduced)